kafka_utils/kafka_manual_throttle/main.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+
+from kafka_utils.util import config
+from kafka_utils.util.zookeeper import ZK
+
+
+LEADER_THROTTLE_RATE_CONFIGURATION = "leader.replication.throttled.rate"
+FOLLOWER_THROTTLE_RATE_CONFIGURATION = "follower.replication.throttled.rate"
+
+
+def parse_opts():
+    parser = argparse.ArgumentParser(
+        description='Manually applies throttle values for brokers in a Kafka cluster',
+    )
+    parser.add_argument(
+        '--cluster-type',
+        '-t',
+        required=True,
+        help='cluster type, e.g. "standard"',
```

Comment:
The examples for this and the name are Yelp-isms, not sure if we want to keep them or not.

Reply:
This was copied from [kafka-rolling-restart](https://github.com/Yelp/kafka-utils/blob/9943ffb35c331eb5b0db4fc61e7175fd3a92e49b/kafka_utils/kafka_rolling_restart/main.py#L66). At this point, it probably does not hurt have consistent examples across all the commands